pipelines/rj_cvl__osinfo_mongo/tasks.py

```
@@ -0,0 +1,502 @@
+    """
+    # Read parquet with chunks
+    chunks_df = pd.read_parquet(temp_file_path)
+
+    # Sort by chunk order 'n'
+    chunks_df = chunks_df.sort_values('n')
+
+    # Concatenate bytes from 'data' column
+    pdf_bytes = b''.join(chunks_df['data'].values)
+
+    # Save PDF to temporary file
+    pdf_temp_file = Path(f"/tmp/pdf_{file_id}.pdf")
```

Ruff (0.14.8)

Probable insecure usage of temporary file or directory: "/tmp/pdf_"

(S108)